apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -210,8 +210,7 @@ ${collectionName}`);
                                    payload.ciphertext)
           .then(recordArrayBuffer => {
         var recordObj;
-        const recordJSON = String.fromCharCode.apply(
-            null,
+        const recordJSON = StringConversion.byteArrayToUtf16String(
```

Comment:
Or maybe we should be using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint? (just found out about that)